cgo.go

```diff
@@ -0,0 +1,11 @@
+// +build !customenv
+
+package gocv
+
+/*
+#cgo !windows pkg-config: opencv
+#cgo darwin   CXXFLAGS:   --std=c++1z -stdlib=libc++
+#cgo windows  CPPFLAGS:   -IC:\opencv\build\install\include
+#cgo windows  LDFLAGS:    -LC:\opencv\build\install\x64\mingw\lib -lopencv_core341 -lopencv_face341 -lopencv_videoio341 -lopencv_imgproc341 -lopencv_highgui341 -lopencv_imgcodecs341 -lopencv_objdetect341 -lopencv_features2d341 -lopencv_video341 -lopencv_dnn341 -lopencv_xfeatures2d341 -lopencv_plot341 -lopencv_tracking341 -lopencv_img_hash341
```

Comment:
That fact that the version number is now hardcoded here makes the upgrade process much more opaque. I can certainly imagine someone rolling the default opencv version in the makefile, pushing that without testing on Windows and forgetting to update these values.

Reply:
The Windows flags were copied directly from `env.cmd` I didn't notice the hardcoded version number.

Do you know if there's a better way to go about this for Windows? Windows could be omitted, but to me that seems less friendly to users than providing some defaults that can be overridden.

Hopefully AppVeyor would catch a mismatch with the default version. Probably a good idea to remove explicitly setting the `CGO_*` variables in `appveyor.yaml` and `.travis.yml`

Reply:
Too bad.